Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -351,35 +302,61 @@ public extension UIView {
         }
     }
 
-    /// SwifterSwift: Rotate view by angle on relative axis.
+    /// SwifterSwift: Rotate view by radians on relative axis.
     ///
     /// - Parameters:
-    ///   - angle: angle to rotate view by.
-    ///   - type: type of the rotation angle.
-    ///   - animated: set true to animate rotation (default is true).
+    ///   - radian: radians to rotate view by.
+    ///   - animated: set true to animate rotation (default is false).
     ///   - duration: animation duration in seconds (default is 1 second).
     ///   - completion: optional completion handler to run with animation finishes (default is nil).
-    func rotate(byAngle angle: CGFloat, ofType type: AngleUnit, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
-        let angleWithType = (type == .degrees) ? .pi * angle / 180.0 : angle
+    func rotate(byRadian radian: CGFloat, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
         let aDuration = animated ? duration : 0
         UIView.animate(withDuration: aDuration, delay: 0, options: .curveLinear, animations: { () -> Void in
-            self.transform = self.transform.rotated(by: angleWithType)
+            self.transform = self.transform.rotated(by: radian)
         }, completion: completion)
     }
 
-    /// SwifterSwift: Rotate view to angle on fixed axis.
+    /// SwifterSwift: Rotate view by degrees on relative axis.
     ///
     /// - Parameters:
-    ///   - angle: angle to rotate view to.
-    ///   - type: type of the rotation angle.
+    ///   - degree: degrees to rotate view by.
     ///   - animated: set true to animate rotation (default is false).
     ///   - duration: animation duration in seconds (default is 1 second).
     ///   - completion: optional completion handler to run with animation finishes (default is nil).
-    func rotate(toAngle angle: CGFloat, ofType type: AngleUnit, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
-        let angleWithType = (type == .degrees) ? .pi * angle / 180.0 : angle
+    func rotate(byDegree degree: CGFloat, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
```

Comment:
Same thing here about making it pluralised
```suggestion
    func rotate(byDegrees degrees: CGFloat, animated: Bool = false, duration: TimeInterval = 1, completion: ((Bool) -> Void)? = nil) {
```

Reply:
Same for the other functions...